src/main/kotlin/no/nav/familie/ba/sak/common/http/interceptor/AutoriserInterceptor.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.ba.sak.common.http.interceptor
+
+import no.nav.familie.ba.sak.behandling.steg.BehandlerRolle
+import no.nav.familie.ba.sak.config.RolleConfig
+import no.nav.familie.ba.sak.sikkerhet.SikkerhetContext
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Import
+import org.springframework.stereotype.Component
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+@Component
+@Import(RolleConfig::class)
+class AutoriserInterceptor(private val rolleConfig: RolleConfig) : HandlerInterceptorAdapter() {
```

Comment:
```suggestion
class RolletilgangInterceptor(private val rolleConfig: RolleConfig) : HandlerInterceptorAdapter() {
```
Litt vagt navn på denne interceptoren, hva med noe slikt?